tests/e2e/specs/wcpay/merchant/merchant-disputes-view-details-via-order-notice.spec.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { fillCardDetails, setupProductCheckout } from '../../../utils/payments';
+
+const { merchant, shopper } = require( '@woocommerce/e2e-utils' );
+
+describe( 'Disputes > View dispute details via disputed order notice', () => {
+	beforeAll( async () => {
+		await page.goto( config.get( 'url' ), { waitUntil: 'networkidle0' } );
```

Comment:
Curious what this `url` is, I couldn't find the `config` module. Is it the Woo Admin dashboard url? Would be nice if the name was clearer (non blocking, asking so I can better understand the e2e setup/config).

Reply:
Ah found it, it's the root host / site URL, in [test.json](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e/config/test.json).